tools/uwuw_preproc

```diff
@@ -496,35 +544,34 @@ def parsing():
     if not args.nuc_data:
         raise Exception('nuc_data file path not specified!!. [-d] is not set')
     if not args.output:
-        args.output = 'output.h5m'
+        args.output = args.datafile
     return args
 
 """
 main
 """
+
+
 def main():
```

Comment:
Typically, all the option parsing should go in `main()`. Everything you have in main should go in another function (called by `main()`) named `uwuw_preproc()`. That way you can access all the capabilities of the script by calling the `uwuw_preproc()` function from Python.